tests/build-and-run-tests.sh

```diff
@@ -234,8 +234,10 @@ function main {
 
     DOC_CHANGE_PATTERN="\
             -e ^Documentation/ \
-            -e ^(README|ROADMAP|CONTRIBUTING|CHANGELOG)$ \
```

Comment:
non-MarkDown files (MAINTAINERS,LICENSE,DCO) are already catched separately, further in the diff.